tools/cli/commands/test.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * External dependencies
+ */
+import chalk from 'chalk';
+import inquirer from 'inquirer';
+import child_process from 'child_process';
+import path from 'path';
+
+/**
+ * Internal dependencies
+ */
+import promptForProject from '../helpers/promptForProject';
+import { readComposerJson } from '../helpers/json.js';
+import { allProjects } from '../helpers/projectHelpers';
+
+/**
+ * Command definition for the test subcommand.
+ *
+ * @param {object} yargs - The Yargs dependency.
+ * @returns {object} Yargs with the build commands defined.
+ */
+export function testDefine( yargs ) {
+	yargs.command(
+		'test [project] [test]',
+		'Runs tests for a specific project.',
+		yarg => {
+			yarg
+				.positional( 'project', {
+					describe:
+						'Project in the form of type/name, e.g. plugins/jetpack, or type, e.g. plugins.',
+					type: 'string',
+				} )
+				.positional( 'test', {
+					alias: 't',
+					describe: 'The test to run',
+					type: 'Array',
+					choices: [ 'js', 'php', 'coverage' ],
```

Comment:
As with the comment below, it'd be nice not to restrict the possible tests here if we're not doing so in `promptForTest()`.